reference/docs-conceptual/community/contributing/editorial-checklist.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Editorial checklist
+description: This is a summarized list of rules for editing PowerShell documentation.
+ms.date: 03/05/2020
+ms.topic: conceptual
+---
+# Editor's checklist
+
+This is a summary of rules to apply when writing new or updating existing articles. See other
+articles in the Contributor's Guide for detailed explanations and examples of these rules.
+
+## Metadata
+
+- `ms.date`: must be in the format **MM/DD/YYYY**
+  - Change the date when there is a significant or factual update
+    - Reorganizing the article
+    - Fixing factual errors
+    - Adding new information
+  - Do not change the date if the update is insignificant
+    - Fixing typos and formatting
+- `title`: unique string of 43-59 chars including spaces
+  - Do not include site identifier (it is auto-generated)
+  - Use sentence case - capitalize only the first word and any proper nouns
+- `description`: 115-145 characters including spaces - this abstract displays in the search result
+
+## Formatting
+
+- Backtick syntax elements that appear, inline, within a paragraph
+  - Cmdlet names `Verb-Noun`
+  - Variable `$counter`
+  - Syntactic examples `Verb-Noun -Parameter`
+  - File paths `C:\Program Files\PowerShell`, `/usr/bin/pwsh`
+  - URLs that are not meant to be clickable in the document
+- Use bold for property names, parameter values, parameter names, class names, module names, entity
+  names, object or type names
+  - Bold is used for semantic markup, not emphasis
+  - Bold - use asterisks `**`
+- Italic - use underscore `_`
+  - Only used for emphasis, not for semantic markup
+- Line breaks at 100 columns (or at 80 for **about_Topics**)
+- No hard tabs - use spaces only
+- No trailing spaces on lines
+
+### Headers
+
+- H1 is first - only one H1 per article
+- Use [ATX Headers](https://github.github.com/gfm/#atx-headings) only
+- Use sentence case for all headers
+- Max depth of H3 or H4
+- Blank line before and after
+- PlatyPS enforces specific headers in its schema - do not add or remove headers
+
+### Code blocks
+
+- Blank line before and after
+- Use tagged code fences - **powershell**, **Output**, or other appropriate language ID
+- Untagged fence - syntax blocks or other shells
+- Put **Output** in separate code block except for simple examples where you don't intend the for
+  the reader to use the **Copy** button
+
+### Lists
+
+- Indented properly
+- Blank line before first item and after last item
+- Bullet - use hyphen "-" not asterisk "*" - too easy to confuse with emphasis
```

Comment:
Instead of quotes around those, should use the tick. The asterisk is unescaped. It renders fine, but formats in pure markdown oddly.

Reply:
Fixed